docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,147 @@
+---
+meta:
+  title: "Webhooks"
+  parentTitle: "Guides"
+  description: "Learn Liveblocks Webhooks concepts and implementation"
+---
+
+## Webhooks
+
+In this guide, we will explain how to use webhooks within your product, to ensure that you are able to store and process Liveblocks data real time. Understanding how to implement webhooks can make your mulitplayer experience observable, and reduce development time by eliminating the need for polling.
```

Comment:
```suggestion
In this guide, we will explain how to use webhooks within your product, to ensure that you are able to store and process Liveblocks data in real time. Understanding how to implement webhooks can make your mulitplayer experience observable, and reduce development time by eliminating the need for polling.
```

Reply:
> In this guide, we will explain how to use webhooks within your product so that you can react to Liveblocks events as they happen.

What do you think? This feels simpler

Reply:
> Understanding how to implement webhooks can make your mulitplayer experience observable, and reduce development time by eliminating the need for polling.

Liveblocks is observable, not the customer's platform. So I would suggest something along those lines:

Now that our platform is observable, you can use webhooks to reduce development time and the need for polling.